frontend/app/components/wp-relations/wp-relations-hierarchy-row/wp-relations-hierarchy-row.template.html

```diff
@@ -0,0 +1,41 @@
+<div class="relation-row" ng-mouseover="$ctrl.showRelationControls = true" ng-mouseleave="$ctrl.showRelationControls = false">
+    <div class="grid-block v-align hierarchy-item" ng-if="!$ctrl.showEditForm && $ctrl.relatedType !== 'parent' && $ctrl.relatedWorkPackage">
+        <div class="grid-content medium-6 collapse" wp-single-relation>
+            <span ng-style="{'padding-left': $ctrl.indentBy + 'px'}" class="wp-relations-hierarchy-subject">
+                <a href="{{$singleRelation.workPackagePath($ctrl.relatedWorkPackage.id)}}" title="{{$singleRelation.getFullIdentifier($ctrl.relatedWorkPackage)}}">
```

Comment:
Stop prefixing your names with `$`, as it is reserved for angular internal stuff.
`$ctrl` is the exception, because we mimic angular 1.5 behaviour.